distributed/scheduler.py

```diff
@@ -88,6 +93,12 @@
 except ImportError:
     compiled = False
 
+try:
+    import zeroconf
+    from zeroconf.asyncio import AsyncServiceInfo, AsyncZeroconf
+except ImportError:
+    zeroconf = False
```

Comment:
I was curious about how much this would add to import time

```python
In [1]: import distributed

In [2]: time import zeroconf
CPU times: user 17.6 ms, sys: 7.92 ms, total: 25.5 ms
Wall time: 27.7 ms
```

Not terrible

Reply:
It is pure Python and only has one dependency on `ifaddr` too so it wouldn't be huge to add to distributed's dependencies. I was just being conservative here.